src/components/AttachmentPicker/index.native.js

```diff
@@ -46,7 +46,7 @@ const documentPickerOptions = {
  */
 function getDataForUpload(fileData) {
     return {
-        name: fileData.fileName || 'chat_attachment',
+        name: fileData.name || 'chat_attachment',
```

Comment:
This fixes the issue for the document picker but breaks it for the image picker as both use this function
one would call it with `fileData` that has `fileName` - image-picker results, the other with `fileData` that has just `name` 
This should have been changed to 

```js
name: fileData.fileName || fileData.name || 'chat_attachment'
```

Reply:
I'd recommend creating a new pull request with those changes, and tag the new issue that @kidroca created and then add me as a reviewer on it

Reply:
@thienlnam 
Here is the PR. I can't manually add you to the list of reviewers.
https://github.com/Expensify/Expensify.cash/pull/3408

Thanks